worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -88,6 +92,20 @@ public IngestionBatchWorkerPlugin()
         simpleModule.addDeserializer(DocumentWorkerDocument.class,
                                      new DocumentWorkerDocumentDeserializer(totalSubdocumentLimit.get()));
         mapper.registerModule(simpleModule);
+
+        validationFile = System.getenv("CAF_VALIDATION_FILE");
+        if (StringUtils.isNotEmpty(validationFile)) {
+            try {
+                fieldValidator = new FieldValidator(validationFile);
+            } catch (final ValidationFileAdapterException validationFileAdapterException) {
+                log.error("Exception when attempting to read validation file" + "\n"
+                    + validationFileAdapterException.getMessage());
```

Comment:
Don't customize exception logging.  Use the overload that takes a message and the exception, so that the logging library can handle the formatting of the exception properly.

```suggestion
                log.error("Exception when attempting to read validation file", validationFileAdapterException);
```